demos/browser/app/meetingV2/video/VideoTileCollection.ts

```diff
@@ -42,8 +40,6 @@ const VideoUpstreamMetricsKeyStats: { [key: string]: string } = {
 };
 
 const VideoDownstreamMetricsKeyStats: { [key: string]: string } = {
-  videoDownstreamGoogFrameHeight: 'Frame Height',
-  videoDownstreamGoogFrameWidth: 'Frame Width',
```

Comment:
Will there be a guide on how to migrate to the new SDK version for metrics?

Reply:
Yes, I will add those in a revision later today. But the migration doc will only focus on changes to `metricsDidReceive` itself.

Reply:
Added the migration doc.